Makefile

```diff
@@ -290,7 +299,8 @@ push-all-images:  push-api-image \
                   push-operator-index-image \
                   push-populator-controller-image \
                   push-ovirt-populator-image \
-                  push-openstack-populator-image
+                  push-openstack-populator-image\
+				  push-ova-provider-server-image
```

Comment:
I'm not sure why it's not aligned here, in my IDE everything is in place.

Reply:
The lines above are indented with spaces only. Your line uses mix of tabs and spaces and your IDE seems to be configured to show tab as 4 spaces (the common default is 8 spaces) that's why it aligns when you look at it.

Reply:
it's worked, Thanks!